### ISSUE TITLE
Build a To-Do app for a language not covered yet

### ISSUE THREAD

**Issue by Amruta-Ranade** [cockroachdb/cockroachdb-todo-apps#10]
Code newbies and new open source contributors -- help us build our repository of CockroachDB To-Do apps!

**What is CockroachDB?**
CockroachDB is the distributed SQL database built to scale fast, survive anything, and thrive everywhere. Whether you are a single-person startup or a global enterprise, CockroachDB was designed to make managing data easy. To learn more about CockroachDB, see our [website](https://www.cockroachlabs.com/) and [docs](https://www.cockroachlabs.com/docs/stable/).

**Detailed instructions to help you build the app**
For step-by-step instructions, see the [README](https://github.com/cockroachdb/cockroachdb-todo-apps/blob/master/README.md)

**Additional Resources**
https://www.cockroachlabs.com/docs/v20.1/hello-world-example-apps.html

**Comment by AntoTomAbraham** [comment]
i would like to work for it.i would build it using React

**Comment by Amruta-Ranade** [comment]
Go for it!

**Comment by abhinav-193** [comment]
I would like to work on it. A todo app using c++

**Comment by Amruta-Ranade** [comment]
> I would like to work on it. A todo app using c++

@abhinav-193 We have an issue for that! https://github.com/cockroachdb/cockroachdb-todo-apps/issues/9
Go for it!

**Comment by mmutiso** [comment]
We now have a C# TO-DO app using EF. Check it out.

**Comment by Amruta-Ranade** [comment]
> We now have a C# TO-DO app using EF. Check it out.

🎉

**Comment by sashithaf16** [comment]
Hi! Is this still open ? Didn't see anything using php. can I give a try for it ?

**Comment by Amruta-Ranade** [comment]
> Hi! Is this still open ? Didn't see anything using php. can I give a try for it ?

Yep!

**Comment by sashithaf** [comment]
Awesome 😃 and is it ok to use a mvc framework like codeigniter ?

**Comment by sashithaf16** [comment]
> > Hi! Is this still open ? Didn't see anything using php. can I give a try for it ?
> 
> Yep!

Awesome 😃 and is it ok to use a mvc framework like codeigniter ?

**Comment by Amruta-Ranade** [comment]
Yep..as long as you use CockroachDB, the language/framework is your choice.

**Comment by shivangi942** [comment]
I would like to build a To-do app using JavaScript.

**Comment by Amruta-Ranade** [comment]
Go for it!
